deploy/docker-compose/README.md

```diff
@@ -0,0 +1,98 @@
+# Docker Compose Deployment
+
+## Quick Start
+
+This is a simple example that shows you how to connect `nm-vllm` with production monitoring via Prometheus and Grafana via Docker Compose.
+
+## Install
+
+Make sure you have Docker and Docker Compose installed.
+- [`docker`](https://docs.docker.com/engine/install/)
+- [`docker compose`](https://docs.docker.com/compose/install/linux/#install-using-the-repository)
+
+> Note: `nm-vllm` requires a GPU. Make sure you have installed the [Nvidia Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
+
+## Deploy
+
+### Launch
+
+The following spins up an `nm-vllm` instance, a Prometheus server, and a Grafana server:
+
+```bash
+docker compose up
+```
+
+### Simulate Client Requests
+
+Install requirements for client:
+```bash
+pip install -r client/requirements.txt
+```
+
+Download some sample data:
+```bash
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
```

Comment:
Can we fork this dataset or something to have a URL that is more neuralmagic and more under our control?